External/FEXCore/Source/Interface/Core/CPUID.h

```diff
@@ -13,6 +13,8 @@ namespace Context {
   struct Context;
 }
 
+
+// #define CPUID_AMD
```

Comment:
Leftover debug comment?

Reply:
Added a comment to this line stating that it is for debugging.
Allows switching between which CPU family FEX represents itself as for testing.